expat/CMakeLists.txt

```diff
@@ -53,9 +53,13 @@ option(WARNINGS_AS_ERRORS "Treat all compiler warnings as errors" OFF)
 if(NOT WIN32)
     option(XML_DEV_URANDOM "Define to include code reading entropy from `/dev/urandom'." ON)
     if(XML_DEV_URANDOM)
-        add_definitions(-DXML_DEV_URANDOM)
-    endif()
+        set(XML_DEV_URANDOM 1)
+    else(XML_DEV_URANDOM)
+        set(XML_DEV_URANDOM 0)
+    endif(XML_DEV_URANDOM)
 endif()
+option(XML_UNICODE "Use UTF-16 encoded chars insted of UTF-8" OFF)

```

Comment:
Typo `insted` here. Besides the UTF-16 encoding the fact that characters go two-byte units with this switch would be cool to mention. Without `XML_UNICODE_WCHAR_T` it's `unsigned short`.